sql/src/main/java/io/crate/operation/projectors/writer/OutputFile.java

```diff
@@ -46,7 +45,7 @@ public OutputFile(URI uri, Settings settings) {
     }
 
     @Override
-    public void open() throws IOException {
+    public OutputStream acquireOutputStream() throws IOException {
```

Comment:
maybe add some note that the acquired stream must be closed by the one who acquired it